demo-kubernetes-with-libstorage/README.md

```diff
@@ -0,0 +1,71 @@
+# Exploring Kubernetes with libstorage persistent volumes on AWS - the Hard Way

```

Comment:
hah. i i'll tone down the words. that's my job. you all just worry about giving me clear instructions